packages/react/src/components/Masthead/Masthead.js

```diff
@@ -256,7 +256,7 @@ const Masthead = ({
       if (!matchFound) {
         if (sections.url === currentUrlPath) {
           matchFound = true;
-        } else if (sections?.menuSections[0]) {
+        } else if (sections?.menuSections && sections?.menuSections[0]) {
```

Comment:
I believe we can use optional chaining here too right?

```suggestion
        } else if (sections?.menuSections?.[0]) {
```